crypto/hd/hdpath_test.go

```diff
@@ -213,3 +214,72 @@ func TestCreateHDPath(t *testing.T) {
 		})
 	}
 }
+
+// Tests to ensure that any index value is in the range [0, max(int32)] as per
+// the extended keys specification. If the index belongs to that of a hardened key,
+// its 0x80000000 bit will be set, so we can still accept values in [0, max(int32)] and then
+// increase its value as deriveKeyPath already augments.
+// See issue https://github.com/cosmos/cosmos-sdk/issues/7627.
+func TestDeriveHDPathRange(t *testing.T) {
```

Comment:
I wonder why the hd package doesn't support paths with the `m/` prefix?

Reply:
I am unfamiliar with these packages unfortunately :)